google/cloud/security/common/data_access/sql_queries/cleanup_tables_sql.py

```diff
@@ -0,0 +1,28 @@
+# Copyright 2018 The Forseti Security Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+#
+"""SQL queries for management of inventory tables."""
+
+RESOURCE_NAME = 'cleanup_tables'
+
+SELECT_SNAPSHOT_TABLES_OLDER_THAN = """
+    SELECT tables.TABLE_NAME 'table'
+    from information_schema.tables as tables
+    inner join snapshot_cycles as snap
+    ON snap.complete_time < DATE_SUB(NOW(), INTERVAL %s DAY)
```

Comment:
`complete_time`is not always populated; like when someone halts the inventory process in mid-stream.  It might be better to change to `snap.start_time`.

What happens if ```0```is passed into ```INTERVAL %s DAY```? Will it work for Angelo's use case of deleting all previous tables?

Reply:
Good call out, will look into this - I think I will need to move the call earlier in the execution of inventory and do some testing, to make sure I don't inadvertently delete the run that just completed!